src/lib/images.ts

```diff
@@ -201,12 +203,10 @@ function stringifyOptions(options: CloudflareImageOptions): string {
     }, '');
 }
 
+// Reused buffer for FNV-1a hashing
+const fnv1aUtf8Buffer = new Uint8Array(100);
```

Comment:
Is `100` size enough?

Reply:
We can probably increase this length slightly since images URLs are usually closer to 200 something characters 👍 
It's not mandatory as described [here](https://github.com/sindresorhus/fnv1a/blob/main/index.d.ts#L14), but worth it as long as we're using it.